resolver/resolver_test.go

```diff
@@ -351,6 +351,38 @@ func TestHTTP(t *testing.T) {
 	}
 }
 
+// TestHTTPAcceptApplicationJson tests that valid requests that specify
+// 'Accept: application/json' succeed. This used to fail with
+// 406 Not Acceptable.
+// See https://jira.mesosphere.com/browse/DCOS_OSS-611
+func TestHTTPAcceptApplicationJson(t *testing.T) {
+	// setup DNS server (just http)
+	res, err := fakeDNS()
+	if err != nil {
+		t.Fatal(err)
+	}
+	res.version = "0.1.1"
```

Comment:
Good question - copy-pasted from `TestHTTP`. I suspect it is to make the tests proof against future API version updates.